### PR TITLE
Remove unneeded ble dependency.

### PR DESCRIPTION
**Pull request by mattThousand**
**Why:**
We would like the apps to not request permissions/entitlements that aren't needed for intended functionality. Because the `Exposure Notifications` APIs eliminate the need for our own ble implementation, we can remove the related dependencies. 

**This commit:**
Removes `react-native-ble-advertiser` and code that imports or makes use of it.

### CODE REVIEW

**Review by tparesi — DISMISSED** [review]
Thanks for doing this. LGTM

File: app/views/LocationTracking.js [hunk 1]
```diff
@@ -246,10 +245,6 @@ class LocationTracking extends Component {
   }
 
   willParticipate = () => {
-    SetStoreData(PARTICIPATE, 'true').then(() => {
```
Comment (tstirrat) [inline app/views/LocationTracking.js]: This is a functional change that might be needed. revert for now.

**Review by tstirrat — DISMISSED** [review]

**Review by tstirrat — DISMISSED** [review]

**Review by johnschoeman — DISMISSED** [review]
💯

**Comment by tstirrat** [comment]
The e2e tests are timing out. is that new to this PR?

File: e2e/pages/Onboarding3.po.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Onboarding3 {
   async isOnScreen(languageStrings) {
     // eslint-disable-next-line jest/no-standalone-expect
     await expect(
-      element(by.label(languageStrings.label.launch_screen3_header)),
+      element(by.label(languageStrings.label.launch_screen3_header_location)),
```
Comment (tstirrat) [inline e2e/pages/Onboarding3.po.js]: not related to this change. fix these in a different PR

**Review by tstirrat — DISMISSED** [review]

**Review by tstirrat — DISMISSED** [review]

**Review by tstirrat — APPROVED** [review]

**Pull request merged**